### PR TITLE
Feature/player move to side

### PR DESCRIPTION
**Pull request by RyanDefost**

### CODE REVIEW

File: Assets/Scripts/TransportMoving/TransportTrigger.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+using UntitledCube.WorldRotation;
+
+namespace UntitledCube.Transport
+{
+    public class TransportTrigger : MonoBehaviour
+    {
+        [SerializeField] private WorldRotations _transportSide;
+        [SerializeField] private LayerMask[] _TransportableLayer;
```
Comment (swzwij) [inline Assets/Scripts/TransportMoving/TransportTrigger.cs]:
```suggestion
        [SerializeField] private LayerMask[] _transportableLayer;
```

File: Assets/Scripts/WorldRotation/WorldRotator.cs [hunk 2]
```diff
@@ -8,22 +8,22 @@ namespace UntitledCube.WorldRotation
...
-        private Dictionary<RotationDirection, Vector3> _directions = new Dictionary<RotationDirection, Vector3>()
+        private Dictionary<WorldRotations, Vector3> _directions = new Dictionary<WorldRotations, Vector3>()
         {
-            { RotationDirection.UP, new Vector3(-90f, 0f, 0f) },
-            { RotationDirection.DOWN, new Vector3(90f, 0f, 0f) },
-            { RotationDirection.LEFT, new Vector3(0f, 90f, 0f) },
-            { RotationDirection.RIGHT, new Vector3(0f, -90f, 0f) }
+            { WorldRotations.UP, new Vector3(-90f, 0f, 0f) },
+            { WorldRotations.DOWN, new Vector3(90f, 0f, 0f) },
+            { WorldRotations.LEFT, new Vector3(0f, -90f, 0f) },
```
Comment (swzwij) [inline Assets/Scripts/WorldRotation/WorldRotator.cs]:
```suggestion
            { WorldRotations.LEFT, new Vector3(0f, 270f, 0f) },
```
Nit pick but I personaly prefer not using minus rotations. Do please check if it still works

**Review by swzwij — CHANGES_REQUESTED** [review]
1. Does it work on phone.
2. Can you implement it into the develop scene

File: Assets/Scripts/TransportMoving/TransportMover.cs [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+    [Serializable]
+    public struct OffsetDirection
+    {
+        [SerializeField] private WorldRotations _side;
+        [SerializeField] private Vector3 _offset;
+
+        public readonly WorldRotations Side => _side;
+        public readonly Vector3 Offset => _offset;
+    }
```
Comment (RyanDefost) [inline Assets/Scripts/TransportMoving/TransportMover.cs]: Should this be in the same script as TransportMover?

**Review by swzwij — APPROVED** [review]

**Pull request merged**